external/icu_decls/unicode/platform.h

```diff
@@ -148,6 +148,8 @@
 #   include <TargetConditionals.h>
 #   if defined(TARGET_OS_IPHONE) && TARGET_OS_IPHONE  /* variant of TARGET_OS_MAC */
 #       define U_PLATFORM U_PF_IPHONE
+#   elif defined(TARGET_OS_TV) && TARGET_OS_TV  /* variant of TARGET_OS_MAC */
```

Comment:
It looks like this header is only used on Windows, why is this change needed?

Reply:
I will back out this change and make sure tvOS still builds. It appeared to me, in reading this file, that it was not only used on Windows, since it referenced Apple defines.